src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -644,6 +582,57 @@ public BlockGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 		pane2.setOpaque(false);
 		pane2.add("Center", PanelUtils.totalCenterInPanel(sbbp2));
 
+		JPanel northPanel = new JPanel(new GridLayout(1, 2, 10, 2));

```

Comment:
I would suggest moving this part of the code (list of boxes) to a new class and reuse it for both plant and gui. Then for the plant, add another panel to hold an additional checkbox to enable the box selector.

Similar to how BiomeGUI uses JSpawnEntriesList to define this logic.